configure.in

```diff
@@ -136,76 +128,72 @@ if test "$enable_openssl" = "yes"; then
              [AC_MSG_FAILURE([can't find openssl >1.0.1 crypto lib])])
```

Comment:
I can only speculate that it's formulated that way because OpenSSL releases tack on a letter at the end. The first 1.0.1 release was 1.0.1a, and therefore in some ways > 1.0.1.

In this case, it's just the error message, but I preserved it when altering the check itself in 333fa84e8e because even the pkg-config file for libcrypto has the letter tacked on.

I don't feel comfortable changing it while just doing configure cleanups/improvements as I don't know enough about the requirements. It could be a purely semantic issue, or it could be a real requirement. Others more familiar with the code that uses it would have to comment.